### PR TITLE
Remove unsupported packages

### PR DESCRIPTION
**Pull request by jgleissner**
This PR removes packages that are not supported (anymore) or were not released. Details:

- Remove rsyslog and wget from SLE Micro
- Remove yast2-trans_en_US from 15 SP2+
- Remove vlan (still supported via legacy module but simply obsolete)
- Adjust for yast2-schema successors yast2-schema-default and yast2-schema-micro
- Remove a few packages from SAPCAL that were not released
- Remove a few packages from SAPCAL SP3+ there were dropped from SLE
- Remove not-for-install system-role-sles4sap from SAP module

### CODE REVIEW

**Comment by jgleissner** [comment]
Fixes https://github.com/SUSE-Enceladus/keg-recipes/issues/177.

File: data/products/chost/sle15/packages.yaml [hunk 1]
```diff
@@ -18,12 +18,13 @@ packages:
       - nfs-client
       - open-iscsi
       - pciutils
+      - rsyslog
       - runc
       - socat
       - supportutils-plugin-suse-public-cloud
       - tcpdump
       - vim
-      - vlan
```
Comment (rjschwei) [inline data/products/chost/sle15/packages.yaml:26]: Please split the vlan removal into a separate PR. What replaces vlan, how does k8s setup networking and does SAP (Gardener) use anything form the vlan package are questions we need to answer first and I think we should not block the other changes while those questions get answered.

**Review by rjschwei — CHANGES_REQUESTED** [review]
The sapcal changes also need to be split out, we need to check with SAP first

File: data/products/chost/sle15/packages.yaml (hunk 1 above)
Comment (jgleissner) [reply data/products/chost/sle15/packages.yaml:26]: Can do, but not sure it's worth it. Quoting the vlan website (http://www.candelatech.com/~greear/vlan.html, last change 2005):

> 802.1Q VLAN code is now part of the official kernel, and has been for years and years. MAC-VLAN code has been added since around 2.6.29. It is very unlikely that you need to download anything from this site, the packages are left here for posterity's sake.
> 802.1Q VLANs and MAC-VLANs can be created with the 'ip' utility (vconfig works for vlans, but is crufty and deprecated).

**Comment by jgleissner** [comment]
I've removed the changes for SAPCAL and the removal of 'vlan'. I'll create separate PRs for those.

File: data/products/chost/sle15/packages.yaml (hunk 1 above)
Comment (rjschwei) [reply data/products/chost/sle15/packages.yaml:26]: OK, if it is part of the kernel and the kernel also delivers the user space tools, please double check, then we are OK with this drop.

Comment (jgleissner) [reply data/products/chost/sle15/packages.yaml:26]: The `vlan` package contains the `vconfig` user space tool which is about as deprecated as `ifconfig`.

Comment (rjschwei) [reply data/products/chost/sle15/packages.yaml:26]: I understand that. The question is if you are willing to cause production issues, and deal with the inevitable escalations, at SAP because vconfig is obsolete and gardner may or may not be using it? If you are willing to go down that route without checking then I am OK with merging this. From my point of view it is better to ask and then follow up with a separate PR after we know that we will not break anything.

Comment (jgleissner) [reply data/products/chost/sle15/packages.yaml:26]: Sure, that's why I dropped the removal from the PR.

**Comment by jgleissner** [comment]
So, are we ok to merge?

**Review by rjschwei — APPROVED** [review]
